tdrs-backend/tdpservice/reports/urls.py

```diff
@@ -2,11 +2,23 @@
 
 from rest_framework.routers import DefaultRouter
 from . import views
+from django.urls import path
 
 router = DefaultRouter()
 
 router.register("", views.ReportFileViewSet)
 
-urlpatterns = []
+urlpatterns = [
+    path(
+        "years/<int:stt>",
+        views.GetYearList.as_view(),
+        name="get-year-list-admin",
+    ),
+    path(
+        "years",
```

Comment:
Is this supposed to be defined with and without the stt path? That appears to be required by the view.

Reply:
As discussed in Dev office hours - we need to add a test here to cover the case where no STT is passed as part of the URL path.